src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/model/mapper/EnvelopeMapperTest.java

```diff
@@ -17,46 +17,85 @@
 import uk.gov.hmcts.reform.bulkscanprocessor.validation.model.OcrValidationWarnings;
 
 import java.time.Instant;
-import java.util.Arrays;
 import java.util.List;
 import java.util.Optional;
 
+import static java.util.Arrays.asList;
 import static java.util.stream.Collectors.toList;
 import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
 import static uk.gov.hmcts.reform.bulkscanprocessor.helper.EnvelopeCreator.getEnvelopeFromMetafile;
 import static uk.gov.hmcts.reform.bulkscanprocessor.model.common.DocumentSubtype.COVERSHEET;
 import static uk.gov.hmcts.reform.bulkscanprocessor.model.common.DocumentSubtype.SSCS1;
 import static uk.gov.hmcts.reform.bulkscanprocessor.model.common.DocumentSubtype.WILL;
+import static uk.gov.hmcts.reform.bulkscanprocessor.model.common.DocumentType.CHERISHED;
+import static uk.gov.hmcts.reform.bulkscanprocessor.model.common.DocumentType.FORM;
+import static uk.gov.hmcts.reform.bulkscanprocessor.model.common.DocumentType.OTHER;
 
 public class EnvelopeMapperTest {
 
     @Test
-    public void should_map_zip_envelope_correctly() throws Exception {
-        InputEnvelope zipEnvelope = getEnvelopeFromMetafile();
+    public void should_map_zip_envelope_without_scannable_items_correctly() throws Exception {
```

Comment:
not sure why we need this test case ? Envelope mapper is generic previous one was correct